src/Services/PageResolver.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DrewRoberts\Blog\Services;
+
+use DrewRoberts\Blog\Exceptions\UnresolvedPage;
+use DrewRoberts\Blog\Models\Page;
+
+class PageResolver
```

Comment:
Have no clue what this class is trying to do.  Again, looks like location specific code that was copied here that should be removed.